### ISSUE TITLE
Scroll to bottom with ESC key

### ISSUE THREAD

**Issue by Pytal** [nextcloud/spreed#9260]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Is your feature request related to a problem? Please describe.**
As a user it would be nice to scroll to the bottom of the chat using the <kbd>ESC</kbd> key, in addition to having the button on the bottom-right

**Describe the solution you'd like**
Have the <kbd>ESC</kbd> key trigger scroll to bottom

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by nickvergessen** [comment]
`ESC` is already used for:
1. Exiting full screen mode
2. Unfocussing the chat input

From my POV it would:
1. Be surprising to scroll all to the bottom (never saw this anywhere else before)
2. I can see this happening by accident when trying to achieve the 2. from above

So I would vote for  "Does not need a keyboard shortcut".

**Comment by Pytal** [comment]
> 1. Be surprising to scroll all to the bottom (never saw this anywhere else before)

Saw this in Discord for example, some other apps might have this as well but otherwise just a nice-to-have :)

**Comment by ShGKme** [comment]
That was a surprise to me, that Discord has this feature. Seems to be not very intuitive. Are there any other examples of such approach in a messenger app?

I would prefer to close/deselect the current opened chat by <kbd>ESC</kbd> (open the `/apps/spreed` page). Semantics of the <kbd>ESC</kbd> is to close/quit something.

By the way, a chat can be scrolled to the end by <kbd>END</kbd> button.

**Comment by jancborchardt** [comment]
Yeah, esc to scroll to bottom also seems strange to me. @Pytal is it a common shortcut you saw elsewhere, or what do others use?

**Comment by Pytal** [comment]
> That was a surprise to me, that Discord has this feature. Seems to be not very intuitive. Are there any other examples of such approach in a messenger app?

> Yeah, esc to scroll to bottom also seems strange to me. @Pytal is it a common shortcut you saw elsewhere, or what do others use?

Saw it work that way on https://discord.com and https://app.element.io @ShGKme @jancborchardt

> By the way, a chat can be scrolled to the end by END button.

👌 didn't know that :D

**Comment by Pytal** [comment]
> That was a surprise to me, that Discord has this feature. Seems to be not very intuitive. Are there any other examples of such approach in a messenger app?

> Yeah, esc to scroll to bottom also seems strange to me. @Pytal is it a common shortcut you saw elsewhere, or what do others use?

Saw it work that way on https://discord.com and https://app.element.io @ShGKme @jancborchardt

> By the way, a chat can be scrolled to the end by END button.

👌 didn't know that :D
